### PR TITLE
FlyingF4 extended support: ADC / Battery, UART4 and UART5

### PR DESCRIPTION
**Pull request by murix**
Hi,
This patchset enable following FlyingF4 components:
ADC in PC1 and PC2 pins (battery monitor can be enabled now)
UART4-RX in PC11 pin.
UART5-RX in PD2 pin.

Support UART4 and UART5 in GCS

### CODE REVIEW

**Comment by peabody124** [comment]
jenkins: test this please

**Comment by peabody124** [comment]
jenkins: add to whitelist

**Comment by peabody124** [comment]
jenkins: test this please

File: flight/targets/flyingf4/board-info/pios_board.h [hunk 1]
```diff
@@ -223,7 +223,27 @@ extern uintptr_t pios_com_debug_id;
 //-------------------------
 #define PIOS_ADC_SUB_DRIVER_MAX_INSTANCES       3
 
-#define VREF_PLUS				3.0f
+// PIOS_ADC_PinGet(0) = PC_11
+// PIOS_ADC_PinGet(1) = PC_12
+//-------------------------
+#define PIOS_DMA_PIN_CONFIG                                                                   \
```
Comment (mlyle) [inline flight/targets/flyingf4/board-info/pios_board.h]: backslashes either right next to right edge or aligned?  (tabs instead of spaces?)

**Comment by mlyle** [comment]
@murix - This looks great to me-- have you tested the extra UARTs and the ADC?

**Comment by murix** [comment]
@mlyle  yes, I'm tested  ADC using a APM Power Module connected. 
UART4 and UART5 are RX-only, I'm tested connecting GPS-tx in nmea mode.

File: flight/targets/flyingf4/board-info/pios_board.h (hunk 1 above)
Comment (murix) [inline flight/targets/flyingf4/board-info/pios_board.h]: @mlyle  I'm not understand. 
Is needed reformat code? 
What better way to do it?

Comment (mlyle) [inline flight/targets/flyingf4/board-info/pios_board.h]: Sorry, can you please either align all the backslashes using tabs, or just put them at the end of line?  It's a little ugly how they're on the right and don't line up.  After that, I'll be glad to merge.

Thanks for the additional F4 support!

**Comment by murix** [comment]
@mlyle I'm fixed alignment of backslashes. 
show different results for same text. 
Can you verify if last edit [65578b1] is correct?

**Comment by mlyle** [comment]
Looks great.  Merged.

**Pull request merged**